### PR TITLE
hwinfo: update to 23.2

### PR DESCRIPTION
**Pull request by felixonmars**
Topic Description
-----------------

hwinfo: update to 23.2
This version supports LoongArch.

Package(s) Affected
-------------------

hwinfo 23.2

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [ ] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by felixonmars** [comment]
image

Works as expected.

File: app-utils/hwinfo/autobuild/build [hunk 1]
```diff
@@ -7,3 +7,7 @@ make install \
     GIT2LOG=/usr/bin/true \
     DESTDIR="$PKGDIR" \
     LIBDIR=/usr/lib
+
+abinfo "Moving sbin to bin ..."
+rm -rf "$PKGDIR/sbin"
```
Comment (MingcongBai) [inline app-utils/hwinfo/autobuild/build]: Missing verbosity.

File: app-utils/hwinfo/autobuild/build [hunk 2]
```diff
@@ -7,3 +7,7 @@ make install \
...
+mv "$PKGDIR/usr/sbin" "$PKGDIR/usr/bin"
```
Comment (MingcongBai) [inline app-utils/hwinfo/autobuild/build]: Missing verbosity.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-utils/hwinfo/autobuild/build (hunk 2 above)
Comment (felixonmars) [reply app-utils/hwinfo/autobuild/build]: Both added.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**